samples/system-test/automlVision.test.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright 2018, Google, LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const path = require('path');
+const {assert} = require('chai');
+const execa = require('execa');
+
+const exec = async cmd => (await execa.shell(cmd)).stdout;
+const cmdDataset = `node automl/automlVisionDataset.js`;
+const cmdModel = `node automl/automlVisionModel.js`;
+const cmdPredict = `node automl/automlVisionPredict.js`;
+
+const testDataSetName = `testDataSet`;
+const dummyDataSet = `dummyDataSet`;
+const testModelName = `dummyModel`;
+const testImgPath = `./resources/`;
+const sampleImage2 = path.join(testImgPath, `testImage2.jpg`);
+
+describe(`auto ml vision`, () => {
+  it.skip(`should create, list, and delete a dataset`, async () => {
```

Comment:
Are these all skipped intentionally?

Reply:
@nirupa-kumar @sirtorry are these all long running, e.g. which require model training? Or can any of these actions be performed rather instantaneously?

Reply:
All of them took longer to complete and the time duration had not been consistent. Some of them do require Model training. That was the reason we moved to skipping them. Please let me know if there is anything else we could do.

Reply:
let me rethink the tests